### ISSUE TITLE
InvalidArgumentError (see above for traceback): Found Inf or NaN global norm. : Tensor had NaN values

### ISSUE THREAD

**Issue by aguang1201** [lc222/seq2seq_chatbot_new#21]
每次训练到1000step的时候就报这个错误.
出错是model.py中,下面代码的最后一句:
                optimizer = tf.train.AdamOptimizer(self.learing_rate)
                trainable_params = tf.trainable_variables()
                gradients = tf.gradients(self.loss, trainable_params)
                **_clip_gradients, _ = tf.clip_by_global_norm(gradients, self.max_gradient_norm)_**
貌似是梯度消失吧.把学习率从0.0001改成0.001还是报错.
求指导
